mage_ai/frontend/components/ServerTimeDropdown/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { 
```

Comment:
@anniexcheng Let's move this file to the `mage_ai/frontend/storage` directory and rename the file, since we keep most of our local storage utility methods there.